stories/Button/Guideline.mdx

```diff
@@ -0,0 +1,169 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  Primary,
+  Controls,
+  Unstyled,
+  Source,
+} from '@storybook/blocks';
+import { Button } from '../../src/lib/components/buttonv2/Buttonv2.component';
+
+import * as ButtonStories from './button.stories';
+
+<Meta of={ButtonStories} />
+
+# Button
+
+Buttons are used to trigger an action or event when clicked.
+
+## Size & style
```

Comment:
In general, the size and style part would need some pictures to have a better understanding (it's called "redlining"), such as : https://images.ctfassets.net/7jw9uvgmirvi/7eFu7zc4oXotX8DzxKZy3j/b5a60e6c561528e91b60fd34f2a040d6/5_button_redline.png
It could be in another iteration.

Reply:
I think there is a way to get those from figma "dev" mode. In addition there is a storybook extensions that allow us to include panels from figma, maybe we could use it to import those directly from there ? It however brings some questions because our figma file is private and core-ui public.

Reply:
@JBWatenbergScality I want to extract the Design system from the Figma file, to make it a dedicated file. This one file could be public. 
It makes sense because of:
1. having in the future multiple products fed from the same DS, 
2. performance reason: everything in the same file starts to be too resource-consuming.